modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -71,7 +99,7 @@ spec:
         - /usr/local/bin/kube-scheduler
         - --address=0.0.0.0
```

Comment:
Can be dropped completely as the replacing `--bind-address` defaults to `0.0.0.0`.